README.md

```diff
@@ -121,17 +122,30 @@ You will need to provide your `NEAR_ACCOUNT` AccountId which will be used to rel
 bridgedETH for the `NEAR_RECIPIENT` (this parameter is optional here and only serves for verbose purposes to show the balance of the recipient before and after) <br/>
 Run: `$ make near-finalize-deposit-from-eth-to-evm TX_HASH=<DEPOSIT_TX_HASH_HERE> NEAR_ACCOUNT=<YOUR_NEAR_ACCOUNT_HERE> NEAR_RECIPIENT=<RECIPIENT_HERE>`
```

Comment:
When I ran this I tried without `NEAR_RECIPIENT` since it was supposed to be optional, and I got an error about missing the `eth-recipient`. When I specified with `ETH_RECIPIENT=<RECIPIENT_ADDRESS>` then it worked, so I think this description needs a small update.

Reply:
Thanks! Yeah, as @0x3bfc already mentioned, it should be `ETH_RECIPIENT` instead of `NEAR_RECIPIENT`.